__tests__/app.test.js

```diff
@@ -172,3 +172,82 @@ describe("GET /api/articles/:article_id/comments", () => {
       })
   })
 })
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("201: responds with a new posted comment", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "Northcoders is great.",
+    }
+
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body: { comment } }) => {
+        expect(comment).toEqual(
+          expect.objectContaining({
+            comment_id: expect.any(Number),
+            body: "Northcoders is great.",
+            article_id: 1,
+            author: "butter_bridge",
+            votes: 0,
+            created_at: expect.any(String),
+          })
```

Comment:
Nice work here, good job checking the necessary properties and nice destructuring!